routes/team.js

```diff
@@ -28,4 +29,15 @@ team.post("/data", function(req, res) {
 	});
 });
 
-module.exports = team;
\ No newline at end of file
+team.post("/delete", authHelper.isAdmin, function(req, res) {
```

Comment:
True, but its usually easier to maintain one users endpoint that can be POSTed to and DELETEd to than have separate endpoints doing the services.